how-to-guides/e2e-tracking/notebooks/e2e_tracking.ipynb

```diff
@@ -0,0 +1,749 @@
+    "</a>\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "This notebook shows how you can use Neptune to track a model across all stages of it's lifecycle by\n",
+    "* **Logging model and run metadata to a central project**\n",
+    "* **Comparing runs to select the best performing model**\n",
+    "* **Monitoring a model once in production**\n",
+    "\n",
+    "You will learn how to use the Neptune webapp to explore run metadata, compare runs, and manually promote a model. However, this notebook can also be used as a template to design an automated end-to-end pipeline that covers the entire lifecycle of a model  without needing any manual intervention.\n",
+    "\n",
+    "This example uses Optuna hyperparameter-optimization to simulate training and evaluating multiple scikit-learn models, and Evidently to monitor models in production. However, given Neptune's flexibility and [multiple integrations](https://docs.neptune.ai/integrations/), you can use any library and framework of your choice. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Before you start\n",
+    "\n",
+    "This notebook example lets you try out Neptune anonymously, with zero setup.\n",
+    "\n",
+    "If you want to see the example logged to your own workspace instead:\n",
+    "\n",
+    "  1. Create a Neptune account. [Register &rarr;](https://neptune.ai/register)\n",
+    "  1. Create a Neptune project that you will use for tracking metadata. For instructions, see [Creating a project](https://docs.neptune.ai/setup/creating_project) in the Neptune docs."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install Neptune and dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install -q -U neptune[sklearn,optuna] scikit-learn optuna evidently matplotlib\n",
+    "%pip install -q --user scikit-learn matplotlib"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run this if you get a `RuntimeError: main thread is not in main loop` error\n",
+    "import matplotlib\n",
+    "\n",
+    "matplotlib.use(\"Agg\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Track model training\n",
+    "\n",
+    "In this section, you will\n",
+    "1. Use Optuna to train multiple scikit-learn regression models\n",
+    "2. Leverage Neptune's [scikit-learn](https://docs.neptune.ai/integrations/sklearn/) and [Optuna](https://docs.neptune.ai/integrations/optuna/) integrations to automatically log metadata and metrics to Neptune for easy run comparison, while also using Neptune's [model registry](https://docs.neptune.ai/model_registry/overview/) to track models."
```

Comment:
Maybe separate the MR note into a 3rd list item?

```suggestion
    "In this section, we'll use the following:\n",
    "1. Optuna to train multiple scikit-learn regression models\n",
    "2. Neptune's [scikit-learn](https://docs.neptune.ai/integrations/sklearn/) and [Optuna](https://docs.neptune.ai/integrations/optuna/) integrations to automatically log metadata and metrics to Neptune for easy run comparison\n",
    "3. Neptune's [model registry](https://docs.neptune.ai/model_registry/overview/) to track models"
```

Reply:
"You'll" or "we'll"?
We've referred to the 2nd person everywhere else

Reply:
It's OK to use a mix; depends on the situation. 2nd person is the default when explaining what value the reader can get or what goals they can accomplish, but I often prefer 1st person plural in "pedagogical" statements that hold the reader's hand through a concrete example. Does that makes sense?